src/devices/src/virtio/console/console_control.rs

```diff
@@ -0,0 +1,126 @@
+use std::collections::VecDeque;
+use std::ops::Deref;
+use std::sync::{Arc, Mutex};
+
+use libc::EFD_NONBLOCK;
```

Comment:
Please use `utils::eventfd::EFD_NONBLOCK` instead, which is a wrapper for macOS support.